sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,185 @@
+    array_fields = {field: hl.if_else(hl.len(rows[field]) > 0, rows[field], hl.null(hl.dtype('array<str>')))
+                    for field in STR_ARRAY_FIELDS}
+    rows = rows.annotate(**array_fields)
+
+    rows = rows.rename({'rsid': 'variantId'})
+
+    return rows.key_by('variantId').select(*FIELDS)
+
+
+def main():
+    p = argparse.ArgumentParser()
+    p.add_argument('input_dataset', help='input VCF file')
+    p.add_argument('--matrixtable-path', default='', help='path for Hail MatrixTable data')
+    p.add_argument('--skip-sample-subset', action='store_true')
+    p.add_argument('--ignore-missing-samples', action='store_true')
+    p.add_argument('--project-guid', required=True, help='the guid of the target seqr project')
+    p.add_argument('--gencode-release', type=int, default=29)
+    p.add_argument('--gencode-path', default='', help='path for downloaded Gencode data')
+    p.add_argument('--es-host', default='localhost')
+    p.add_argument('--es-port', default='9200')
+    p.add_argument('--num-shards', type=int, default=6)
+    p.add_argument('--block-size', type=int, default=2000)
+
+    args = p.parse_args()
+
+    if args.matrixtable_path == '':
+        args.matrixtable_path = '{}.mt'.format(os.path.splitext(args.input_dataset)[0])
+
+    hl.init()
+
+    load_gencode(args.gencode_release, genome_version=WGS_SAMPLE_TYPE, download_path=args.gencode_path)
+
+    # For the CMG dataset, we need to do hl.import_vcf() for once for all projects.
+    if os.path.isdir(args.matrixtable_path):
+        reminder = 'If the input VCF file has been changed, or you just want to import VCF again, please delete the MatrixTable.'
+        logger.info('Use the existing MatrixTable at {}. {}'.format(args.matrixtable_path, reminder))
+    else:
+        hl.import_vcf(args.input_dataset, reference_genome='GRCh38').write(args.matrixtable_path)
+        logger.info('The VCF file has been imported to the MatrixTable at {}.'.format(args.matrixtable_path))
+
+    mt = hl.read_matrix_table(args.matrixtable_path)
+
+    start_time = time.time()
+    rows = sub_setting_mt(args.project_guid, mt, WGS_SAMPLE_TYPE, args.skip_sample_subset, args.ignore_missing_samples)
+    logger.info('Variant counts: {}'.format(rows.count()))
+
+    rows = annotate_fields(rows)
+
+    meta = {
+      'genomeVersion': '38',
+      'sampleType': WGS_SAMPLE_TYPE,
+      'datasetType': 'SV',
+      'sourceFilePath': args.input_dataset,
+    }
+    index_name = get_es_index_name(args.project_guid, meta)
+
+    es_password = os.environ.get('PIPELINE_ES_PASSWORD', '')
+    es_client = ElasticsearchClient(host=args.es_host, port=args.es_port, es_password=es_password)
+
+    es_client.export_table_to_elasticsearch(
+        rows,
+        index_name=index_name,
+        index_type_name='_doc',
+        block_size=args.block_size,
+        num_shards=args.num_shards,
+        delete_index_before_exporting=True,
+        export_globals_to_index_meta=True,
```

Comment:
what are the table globals? The `meta` dict defined above is what should actually end up as the index metadata, and this variable tells the ES client to use `table.globals` as the meta in the index, but I don't see you ever setting those globals to the appropriate values